No name 'run_training' in module 'instructlab.training'

For some reason, the latest version of instructlab-training breaks when being imported on some machines.
For example, the CI is currently getting the error:

```
Error: src/instructlab/model/train.py:11:0: E0611: No name 'run_training' in module 'instructlab.training' (no-name-in-module)
```

--- a/src/instructlab/model/train.py
+++ b/src/instructlab/model/train.py
@@ -7,7 +7,7 @@ import os
 import shutil
 
 # Third Party
-# pylint: disable=ungrouped-imports
+# pylint: disable=ungrouped-imports,no-name-in-module
 from instructlab.training import (
     DeepSpeedOptions,
     LoraOptions,

